src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -620,7 +620,7 @@ VirtualRepeatController.prototype.containerUpdated = function() {
         this.repeatListExpression,
         angular.bind(this, function(items) {
           if (items && items.length) {
-            this.$$rAF(angular.bind(this, this.readItemSize_));
+            this.readItemSize_();
```

Comment:
I would love to know why the removal of `$$rAF` fixes this. My first guess is that `this.itemSize` is undefined/null when using `$$rAF`, but defined when not using it.

@crisbeto Do you think you could take a quick dive to figure out what code is/isn't running with this change that might account for the drastic performance increase?

That said, even if we can't figure it out, I still think we should merge this 😄